### PR TITLE
Jnpeterson ce

### PR DESCRIPTION
**Pull request by jeffreynpeterson**
Will need IDs added.

hmm thought I could cherry pick commits differently to make 3 pull requests

### CODE REVIEW

File: topology/Lamar University/FACILITY.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+ID:
```
Comment (matyasselmeci) [inline topology/Lamar University/FACILITY.yaml:1]:
```suggestion
ID: 10185
```

File: topology/Lamar University/Lamar Cluster/Lamar-Cluster.yaml [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+Production: true
+# SupportCenter is one of the support centers in topology/support-centers.yaml
+SupportCenter: Self Supported
+
+# GroupDescription is a long description of the resource group; may be multiple lines.
+GroupDescription: This is a cluster at Lamar University 
+
+# If you have an up-to-date local git clone, fill GroupID with the output from `bin/next_resource_group_id`
+# Otherwise, leave it blank and we will fill in the appropriate value for you.
+GroupID:
```
Comment (matyasselmeci) [inline topology/Lamar University/Lamar Cluster/Lamar-Cluster.yaml:11]:
```suggestion
GroupID: 1111
```

File: topology/Lamar University/Lamar Cluster/Lamar-Cluster.yaml [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+  # Resource Names need to be unique across all resources in the OSG.
+  Lamar-Cluster:
+    # Active is true if the resource is accepting requests, and false otherwise.
+    # When first registering a resource, set this to false. Set it to true when it's ready for production.
+    Active: true
+    # Description is a long description of the resource; may be multiple lines
+    Description: This is a Hosted CE for LamarUniversity.
+    # If you have an up-to-date local git clone, fill ID with the output from `bin/next_resource_id`
+    # Otherwise, leave it blank and we will fill in the appropriate value for you.
+    ID:
```
Comment (matyasselmeci) [inline topology/Lamar University/Lamar Cluster/Lamar-Cluster.yaml:27]:
```suggestion
    ID: 1134
```

File: topology/Lamar University/Lamar Cluster/SITE.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# LongName is the expanded name of the site
+LongName: Lamar University
+# Description is a brief description of your site
+Description: Lamar University Cluster
+
+# If you have an up-to-date local git clone, fill ID with the output from `bin/next_site_id`
+# Otherwise, leave it blank and we will fill in the appropriate value for you.
+ID:
```
Comment (matyasselmeci) [inline topology/Lamar University/Lamar Cluster/SITE.yaml:8]:
```suggestion
ID: 10295
```

File: topology/The College of New Jersey/TCNJ - ELSA/ELSA.yaml [hunk 5]
```diff
@@ -58,3 +58,30 @@ Resources:
           hidden: false
     Tags:
       - CC*
+  TCNJ-ELSA:
```
Comment (matyasselmeci) [inline topology/The College of New Jersey/TCNJ - ELSA/ELSA.yaml:61]: This looks like a duplicate of TCNJ-ELSA from line 34.

File: topology/University of Tennessee Chattanooga/FACILITY.yaml [hunk 6]
```diff
@@ -0,0 +1 @@
+ID:
```
Comment (matyasselmeci) [inline topology/University of Tennessee Chattanooga/FACILITY.yaml:1]:
```suggestion
ID: 10186
```

File: topology/University of Tennessee Chattanooga/UTC - Epyc/SITE.yaml [hunk 7]
```diff
@@ -0,0 +1,27 @@
+# LongName is the expanded name of the site
+LongName: University of Tennessee Chatanooga
+# Description is a brief description of your site
+Description: University of Tennessee Chatanooga Epyc Cluster
+
+# If you have an up-to-date local git clone, fill ID with the output from `bin/next_site_id`
+# Otherwise, leave it blank and we will fill in the appropriate value for you.
+ID:
```
Comment (matyasselmeci) [inline topology/University of Tennessee Chattanooga/UTC - Epyc/SITE.yaml:8]:
```suggestion
ID: 10296
```

File: topology/University of Tennessee Chattanooga/UTC - Epyc/UTC-Epyc.yaml [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+Production: true
+# SupportCenter is one of the support centers in topology/support-centers.yaml
+SupportCenter: Self Supported
+
+# GroupDescription is a long description of the resource group; may be multiple lines.
+GroupDescription: This is a cluster of EPYC processors at University of Tennessee Chatanooga 
+
+# If you have an up-to-date local git clone, fill GroupID with the output from `bin/next_resource_group_id`
+# Otherwise, leave it blank and we will fill in the appropriate value for you.
+GroupID:
```
Comment (matyasselmeci) [inline topology/University of Tennessee Chattanooga/UTC - Epyc/UTC-Epyc.yaml:11]:
```suggestion
GroupID: 1112
```

File: topology/University of Tennessee Chattanooga/UTC - Epyc/UTC-Epyc.yaml [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+  # Resource Names need to be unique across all resources in the OSG.
+  UTC-Epyc:
+    # Active is true if the resource is accepting requests, and false otherwise.
+    # When first registering a resource, set this to false. Set it to true when it's ready for production.
+    Active: true
+    # Description is a long description of the resource; may be multiple lines
+    Description: This is a Hosted CE for University of Tennessee Chatanooga.
+    # If you have an up-to-date local git clone, fill ID with the output from `bin/next_resource_id`
+    # Otherwise, leave it blank and we will fill in the appropriate value for you.
+    ID:
```
Comment (matyasselmeci) [inline topology/University of Tennessee Chattanooga/UTC - Epyc/UTC-Epyc.yaml:27]:
```suggestion
    ID: 1135
```

**Review by matyasselmeci — COMMENTED** [review]
I added IDs. It looks like this PR is adding a second TCNJ-ELSA? That won't work -- the resources are in a dict so one of them will just overwrite the other.

File: topology/The College of New Jersey/TCNJ - ELSA/ELSA.yaml (hunk 5 above)
Comment (jeffreynpeterson) [reply topology/The College of New Jersey/TCNJ - ELSA/ELSA.yaml:61]: The ce was re-deployed with a different more descriptive FQDN.  and the old was set to false.

What is the best way to transition this?

Comment (matyasselmeci) [reply topology/The College of New Jersey/TCNJ - ELSA/ELSA.yaml:61]: Does the old one already have records in GRACC? @djw8605 , is it OK to change the FQDN of an existing resource or would that cause problems?

Comment (djw8605) [reply topology/The College of New Jersey/TCNJ - ELSA/ELSA.yaml:61]: The old one has records in GRACC, and the FQDN was used to match to records.  The old hosted-ce30 needs to stick around for a while.

Comment (matyasselmeci) [reply topology/The College of New Jersey/TCNJ - ELSA/ELSA.yaml:61]: OK. Jeff, you'll need to give the new one a new name (like TCNJ-ELSA2 or something). Use 1137 for the ID.

Comment (matyasselmeci) [reply topology/The College of New Jersey/TCNJ - ELSA/ELSA.yaml:61]: We discussed this in Slack; the existing TCNJ-ELSA entry should be renamed TCNJ-ELSA-old, set inactive, and a new TCNJ-ELSA entry should be created (using the new FQDN, and 1137 for the ID).

**Review by matyasselmeci — APPROVED** [review]

**Pull request merged**